src/promptflow/dev_requirements.txt

```diff
@@ -7,7 +7,6 @@ mock
 nox
 portalocker~=1.2
 pre-commit
-promptflow-tools
```

Comment:
why do we need to remove this?

Reply:
this will install promptflow-tools production version.
This promptflow-tools will install promptflow production version.

We are testing promptflow / promptflow , the code version.

It will take in a lot of irrelavant issues.